test/e2e/common/pod_resize.go

```diff
@@ -0,0 +1,437 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
```

Comment:
nit from the testing side: seems that all of this file would be used for a sig node-spnsored feature.
I think it would be better to have this code in a directory that is specific to sig node.
`/test/e2e/common` should have code that would actually be common to other e2e tests.

Could we move it somewhere else?

Reply:
Thanks for the pointer. I was unsure of this location as well, but found tests for features like empty_dir, runtimeclass etc here. This is a node sponsored feature, but it has a core API change as it adds new fields to Pod Spec & Status, change to kubectl, and a small change to scheduler - so it cuts across the core k8s components. Do you think test/e2e/node is a better location?

Reply:
My initial inclination would be that test/e2e/node would be a bit better (my inclination could be wrong).
Im thinking about the case in which something needs to be done with the test (enhancement or bug fix), what SIG should we ping?

Reply:
It would be SIG-Node, one of the people listed in the OWNERS file would likely be reviewing the e2e test according to my discussion with Dawn during the weekly sig-node meeting a couple of weeks ago.

I looked more closely and saw the runtimeclass feature e2e test living there, this feature is very congruent to runtimeclass. It looks like test/e2e/node is indeed the right place. I can reconfirm with them.